aries-site/src/layouts/content/ExampleControls.js

```diff
@@ -39,12 +39,18 @@ export const ExampleControls = ({ designer, docs, figma, setShowLayer }) => (
     pad={{ horizontal: 'medium', vertical: 'small' }}
     round={{ corner: 'bottom', size: 'small' }}
   >
-    <Box direction="row-responsive" gap="small">
+    <Grid
+      columns={{ count: 'fill', size: ['small', 'auto'] }}
```

Comment:
<img width="1350" alt="Screen Shot 2020-06-08 at 4 02 07 PM" src="https://user-images.githubusercontent.com/12522275/84088574-a9186b00-a9a1-11ea-9631-e798ab0f2455.png">

The "see fullscreen" text is getting forced to two lines even on desktop where there's enough space. Also, on mobile it doesn't seem like there will ever be enough space for both so maybe the entire thing should be single column?
<img width="491" alt="Screen Shot 2020-06-08 at 4 01 49 PM" src="https://user-images.githubusercontent.com/12522275/84088663-df55ea80-a9a1-11ea-89eb-7d526ea2c224.png">